src/java.desktop/share/classes/javax/swing/border/BevelBorder.java

```diff
@@ -142,6 +142,8 @@ public void paintBorder(Component c, Graphics g, int x, int y, int width, int he
      * Reinitialize the insets parameter with this Border's current Insets.
```

Comment:
To be consistent with the superclass:
```suggestion
     * Reinitializes the insets parameter with this Border's current Insets.
```

I guess the entire javadoc can be inherited with {@inheritDoc}, it adds nothing to the text in `AbstractBorder`.